### DISCUSSION TITLE
Card: ALBA

### DISCUSSION THREAD

**Discussion by RebelusQuo** [RebelusQuo/the-wall-is-down#8] [Ideas]
I realize there is more to clarify about this card about simply the "up to" phrase. This is an "east card" which seem to punish US. However, with "up to" US could choose to remove no Influence. The type of Influence is also not limited so US could remove Russian or Chinese Influence. My guess would be the text should be something like:

> US removes 2 [US or West] Influence among Venezuela, Ecuador, Bolivia, Cuba, and Honduras, no more than 1 Influence per country.

**Comment by rodrigoSantamaria** [comment]
Notice that if a West player uses an East card, the player resolving the text is the next East player in the header order. So US will not choose about the influence. By "US loses" I meant the US superpower, not the US player. So the correct wording would be something like:
"Remove up to 2 West Influence among..."

**Comment by RebelusQuo** [comment]
In each place where a superpower is mentioned first I've interpreted this as them having the choice. It can be hard to distinguish between e.g. Influence of a certain superpower and a choice explicitly made by them. Is it worth using the terminology "[power] player" where they are given explicit choice (instead of simply "[power]")?

For this particular card, new suggestion:

> Remove 2 [US or West] Influence from countries among Venezuela, Ecuador, Bolivia, Cuba, and Honduras, no more than 1 Influence per country.

Should I understand your response as a change from only US Influence to West Influence?

**Comment by rodrigoSantamaria** [comment]
Yes, definitely "[power] player" is the way to go to differentiate player from country. 
Sorry, I made a mistake, it is only US influence. "Remove 2 US Influence..."
Thematically, ALBA was mostly about removing US dollar as currency, so it affected US, not UE. Gamewise, EU has few options of entering South America, we should not restrict them more.

**Comment by RebelusQuo** [comment]
Acknowledged.
